src/image/p5.Image.js

```diff
@@ -344,7 +344,7 @@ p5.Image = class {
    */
   _animateGif(pInst) {
     const props = this.gifProperties;
-    const curTime = pInst._lastRealFrameTime;
+    const curTime = pInst._lastRealFrameTime || window.performance.now();
```

Comment:
I think this makes sense. At first I was worried that this wouldn't be as robust as grabbing the time from the main canvas, but there's also the chance that the graphic is being used for fully offscreen rendering and the main canvas never updates, so this is probably the best compromise for now. We can always revisit this later, I'll merge this PR!